Maestro/subscriptions.json

```diff
@@ -184,6 +198,20 @@
             "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'",
             "-DirPropsVersionElements CoreClrExpectedPrerelease"
           ]
+        },
+        // This handler will bring the Latest CoreCLR master build into CoreCLR master
```

Comment:
What in CoreCLR references an earlier version of CoreCLR?  I know why CoreFX needs to be updated with newer CoreFX builds, but what does CoreCLR have a self-dependency?